registrations/tasks.py

```diff
@@ -347,6 +347,23 @@ def set_risk_status(self, registration):
         self.l.info("Identity updated with risk level")
         return risk
 
+    def send_to_jembi(self, registration):
+        """
+        Runs the correct task to send the registration information to
+        Jembi.
+        """
+        authority = BasePushRegistrationToJembi.get_authority_from_source(
+            registration.source)
+        task = {
+            'nurse': push_nurse_registration_to_jembi,
+            'personal': push_registration_to_jembi,
+            'optout': push_registration_to_jembi,
+            'clinic': push_registration_to_jembi,
+            'chw': push_registration_to_jembi,
+        }.get(authority, push_registration_to_jembi)
+
+        return task.delay(registration_id=str(registration.pk))
```

Comment:
I'm not entirely sure what the difference is between `task.delay` and `task.apply_async`. I noticed we're using `apply_async` below.